helm-chart/values.yaml

```diff
@@ -40,4 +40,12 @@ postgres:
   password: postgres
   database: postgres
 
-
+ssrCatalogue:
+  image:
+    repository: molgenis/ssr-catalogue-snapshot
+    tag: latest
```

Comment:
Should we not have explicit version here?

Reply:
latest is good for the default value , but Jenkins ( or other chart user ) should /could set (override) the tag value with the pre-view tag value